src/librustdoc/html/static/main.js

```diff
@@ -1411,6 +1411,14 @@ function getSearchElement() {
             return tmp;
         }
 
+        // In case someone wants to close the current tag to inject some other unwanted things.
+        // For example: "</span><iframe whatever>"
+        function safeHtml(x) {
+            var e = document.createElement('h1');
```

Comment:
Would it be better for this to be a `span` rather than a `h1` ?  While I appreciate the type of the `HTMLElement` subclass isn't entirely relevant, it does look a bit strange here.
```suggestion
            var e = document.createElement('span');
```

Reply:
The point is just to have a "text" node, so either a `h1` or a `span` is the same I assume. But I can change it to `span` if you prefer.

Reply:
Just because the content is to go into a `span` eventually, I'd prefer `span`.